src/components/export/export.js

```diff
@@ -17,19 +17,19 @@ export default class Export extends Component {
         user: new ArgsError("Invalid address", value => ArgsAccount.isValid(value), true),
         dao: new ArgsError("Invalid address", value => ArgsAccount.isValid(value), true),
         multicall: new ArgsError("Invalid address", value => ArgsAccount.isValid(value), true),
-        gas: new ArgsError("Amount out of bounds", value => ArgsNumber.isValid(value)),
+        gas: new ArgsError("Amount out of bounds", value => ArgsBig.isValid(value) && value.value !== ""),
         depo: new ArgsError("Amount out of bounds", value => ArgsBig.isValid(value) && value.value !== ""),
         amount: new ArgsError("Invalid amount", value => ArgsBig.isValid(value) && value.value !== ""),
         token: new ArgsError("Invalid address", value => ArgsAccount.isValid(value)),
         desc: new ArgsError("Invalid proposal description", value => value.value !== "", true),
         noToken: new ArgsError("Address does not belong to token contract", value => this.errors.noToken),
         notWhitelisted: new ArgsError("Token not whitelisted on multicall instance", value => this.errors.notWhitelisted),
-        hasErrors: new ArgsError("You have unfixed errors", value => this.errors.hasErrors)
+        hasErrors: new ArgsError("Please fix all errors", value => this.errors.hasErrors)
     };
 
     total = {
-        gas: new ArgsNumber(toGas(270), 1, toGas(270), "gas"),
-        depo: new ArgsBig("1", "1", null, "yocto"),
+        gas: new ArgsBig("270", toGas("1"), toGas("270"), "Tgas"),
+        depo: new ArgsBig(toNEAR("1"), toNEAR("1"), null, "NEAR"),
```

Comment:
Min and Max are just "numbers", they don't have units, so setting minimum to `toGas("1")` will get you 1e12 as a number. Now your ArgsBig thinks the minimum is 1e24 TGas.
This should be more of what you're looking for
```ts
gas: new ArgsBig("270","1","270","Tgas"), // 1 Tgas as minimum
// or
gas: new ArgsBig("270",toTGas("1"),"270","Tgas"), // 1 Gas as minimum
```
The NEAR one has a default and minimum of 1 yocto, seems alright 👍🏽 

To avoid too many comments, I'll just put this here once, please also correct this in all the cards.

Reply:
My bad, what actually happens is, the first number `value`, will be converted to the lowest unit, then we check it against the boundaries.
Example: 
```ts
new ArgsBig("270", toGas("1"), toGas("270", "Tgas"))
```
`"270"` -- convert --> `Big(270e12)` -- compare --> `Big("1") <= Big(270e12) <= Big("270e12")` --> ✅  no error 

**BUT** there is a different problem here:
```ts
new ArgsBig(toNEAR("1"), toNEAR("1"), null, "NEAR")
```
according to the logic above should really be
```ts
new ArgsBig(toNEAR("1"), "1", null, "NEAR")
```
because otherwise the UI would also accept values < `1e-24` (less than one yocto) when the unit is NEAR.
The reason it does not allow you to do that currently, is because [this](https://github.com/near-multicall/ui/blob/5dc0dd5e0fe6b8218ab0fd479dbfc19dc1d6e6cf/src/utils/args.ts#L157) piece of code saves you.
It makes sure the string inserted, does not have more decimal places than allowed by the unit, in case of NEAR: 24.
Now that we are using Big.js, that piece of code can be removed soon, so we should correct this.

Reply:
Good catch, thank you!